src/lib/mina_lib/coda_subscriptions.ml

```diff
@@ -198,10 +198,12 @@ let create ~logger ~constraint_constants ~wallets ~new_blocks
                    Out_channel.output_lines out_channel
                      [Yojson.Safe.to_string (Lazy.force precomputed_block)] )
            ) ;
-           Option.iter log ~f:(fun `Log ->
-               [%log info] "Saw block $precomputed_block"
-                 ~metadata:[("precomputed_block", Lazy.force precomputed_block)]
-           )) ;
+           [%log info] "Saw block with state hash %s"
```

Comment:
This changes the log message each time, which may make it harder to filter for these. Can we use interpolation instead?